R/pk-parameter-sensitivity.R

```diff
@@ -7,8 +7,8 @@ PKParameterSensitivity <- R6::R6Class("PKParameterSensitivity",
   inherit = DotNetWrapper,
   cloneable = FALSE,
   active = list(
-    #' @field parameterPath  Unique name of parameter in sensitivity analysis
-    parameterPath = function(value) {
+    #' @field parameterName  Unique name of parameter in sensitivity analysis
+    parameterName = function(value) {

```

Comment:
@pchelle @abdullahhamadeh I just realized that what was called parameterPath was actually not a path, but the name as defined in the SA. I have rename this function to be consistent with the rest of the suite. So if you were using explicitly this property, please rename

ALso note that it will already be simplified (normally)
e.g. organism|liver|volume => liver-volume